tests/kokkos/kokkos_utils_tests.cpp

```diff
@@ -544,4 +544,230 @@ TEST_CASE("subviews") {
+                        ++ndiffs;
+                    }
+          }
+          for (int ens = 2; ens < 6; ens++) {
+            k1 = (ens == 2) ? idx0[2] : 0;
+            k2 = (ens == 2) ? idx1[2] : 5;
+            if (ens == 2)
+              testv4 = v4_0;
+            l1 = (ens == 3) ? idx0[3] : 0;
+            l2 = (ens == 3) ? idx1[3] : 4;
+            if (ens == 3)
+              testv4 = v4_1;
+            m1 = (ens == 4) ? idx0[4] : 0;
+            m2 = (ens == 4) ? idx1[4] : 3;
+            if (ens == 4)
+              testv4 = v4_2;
+            n1 = (ens == 5) ? idx0[5] : 0;
+            n2 = (ens == 5) ? idx1[5] : 2;
+            if (ens == 5)
+              testv4 = v4_3;
+            for (int k = k1; k < k2; k++)
+              for (int l = l1; l < l2; l++)
+                for (int m = m1; m < m2; m++)
+                  for (int n = n1; n < n2; n++) {
+                    if (v4(k, l, m, n) !=
+                        testv4(k - k1, l - l1, m - m1, n - n1))
+                      ++ndiffs;
+                  }
+          }
+          for (int ens = 3; ens < 6; ens++) {
+            l1 = (ens == 3) ? idx0[3] : 0;
+            l2 = (ens == 3) ? idx1[3] : 4;
+            if (ens == 3)
+              testv3 = v3_0;
+            m1 = (ens == 4) ? idx0[4] : 0;
+            m2 = (ens == 4) ? idx1[4] : 3;
+            if (ens == 4)
+              testv3 = v3_1;
+            n1 = (ens == 5) ? idx0[5] : 0;
+            n2 = (ens == 5) ? idx1[5] : 2;
+            if (ens == 5)
+              testv3 = v3_2;
+            for (int l = l1; l < l2; l++)
+              for (int m = m1; m < m2; m++)
+                for (int n = n1; n < n2; n++) {
+                  if (v3(l, m, n) != testv3(l - l1, m - m1, n - n1))
+                    ++ndiffs;
+                }
+          }
+          for (int ens = 4; ens < 6; ens++) {
+            m1 = (ens == 4) ? idx0[4] : 0;
+            m2 = (ens == 4) ? idx1[4] : 3;
+            if (ens == 4)
+              testv2 = v2_0;
+            n1 = (ens == 5) ? idx0[5] : 0;
+            n2 = (ens == 5) ? idx1[5] : 2;
+            if (ens == 5)
+              testv2 = v2_1;
+            for (int m = m1; m < m2; m++)
+              for (int n = n1; n < n2; n++) {
+                if (v2(m, n) != testv2(m - m1, n - n1))
+                  ++ndiffs;
+              }
+          }
+          n1 = idx0[5];
+          n2 = idx1[5];
+          testv1 = v1_0;
+          for (int n = n1; n < n2; n++) {
+            if (v1(n) != testv1(n - n1))
+              ++ndiffs;
+          }
+          // Make sure that our diffs counting strategy works
+          // by checking that two entries that should be different
+          // are indeed different.
+          if (v1_0(0) != v6(i0, i1, i2, i3, i4, 1))
+            ++ndiffs;
+        });
+    auto diffs_h = Kokkos::create_mirror_view(diffs);
+    Kokkos::deep_copy(diffs_h, diffs);
+    REQUIRE(diffs_h() == 1);
```

Comment:
Why are you expecting a diff?

Reply:
Haha, that one got me the first time, too.

This came from the original subview tests, and above, at the end of the `parallel_for()`, there's a sanity check to make sure the diffing workflow catches a known diff. So, the count ends up as identically 1 in the passing case